### PR TITLE
Talk remind notification

### PR DESCRIPTION
**Pull request by unasuke**
## what
When the talk will starts soon, send push notification for users who bookmarked the talk.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -38,3 +38,5 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
   end
 end
+
+require_relative "../lib/digested_assets_path_resolver"
```
Comment (okuramasafumi) [inline config/application.rb:42]: zeitwerk did this for us?

File: lib/digested_assets_path_resolver.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require "json"
+
+class DigestedAssetsPathResolver
+  def initialize
+      @manifest_path = File.join("public", "assets", ".manifest.json")
```
Comment (okuramasafumi) [inline lib/digested_assets_path_resolver.rb:5]: should be local var?

File: config/application.rb (hunk 1 above)
Comment (unasuke) [reply config/application.rb:42]: I want to use `autoload_lib(ignore:)` when this app updates to v7.1.

https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#config-autoload-lib-ignore

File: lib/digested_assets_path_resolver.rb (hunk 2 above)
Comment (unasuke) [reply lib/digested_assets_path_resolver.rb:5]: 👍👍 👍

Comment (unasuke) [reply lib/digested_assets_path_resolver.rb]: a36d978

**Review by okuramasafumi — APPROVED** [review]
LGTM!

**Pull request merged**